src/main/java/io/cdap/plugin/successfactors/source/service/SuccessFactorsService.java

```diff
@@ -69,8 +69,8 @@
  */
 public class SuccessFactorsService {
 
-  private static final String TEST = "TEST";
-  private static final String METADATA = "METADATA";
+  public static final String TEST = "TEST";
```

Comment:
can these be package private instead of public?

Reply:
Can't make as it is used outside the package.